examples/rpmsgsocket/rpsock_client.c

```diff
@@ -225,14 +228,14 @@ static int rpsock_unsync_test(struct rpsock_arg_s *args)
   return 0;
 }
 
-static int rpsock_stream_client(int argc, char *argv[])
+static int rpsock_stream_client(FAR char *argv[])
 {
   struct sockaddr_rpmsg myaddr;
   struct rpsock_arg_s args;
   bool nonblock = false;
   int cnt = 0;
-  char *outbuf;
-  char *inbuf;
+  FAR char *outbuf;
+  FAR char *inbuf;
   int sockfd;
   int ret;
 
```

Comment:
What happens in 
```
if (strcmp(argv[2], "nonblock") == 0)
```
when argv[2] is out of bounds? i.e. `argv<2`? will never happen?

Reply:
I know it was here before but maybe worth looking into in a free moment? :-)

Reply:
int main(int argc, char *argv[])
{
  if (argc < 4)
    {
      printf("Usage: rpsock_client stream/dgram"
             " block/nonblock rp_name rp_cpu\n");
      return -EINVAL;
    }

if argc < 4, it will not call rpsock_stream_client, So I don't think it needs to be judged again